test/torch_test.py

```diff
@@ -286,18 +286,74 @@ def test_remote_tensor_iterable_methods(self):
         x = torch.FloatTensor([1, 2, 3]).send(remote)
         y = torch.FloatTensor([2, 3, 4]).send(remote)
         z = torch.FloatTensor([5, 6, 7]).send(remote)
-        x.get()
-        y.get()
-        z.get()
-        assert(torch.equal(torch.stack([x, y, z]), torch.FloatTensor([[1, 2, 3], [2, 3, 4], [5, 6, 7]])))
+
+        assert(torch.equal(torch.stack([x, y, z]).get(), torch.FloatTensor([[1, 2, 3], [2, 3, 4], [5, 6, 7]])))
 
         x = torch.FloatTensor([1, 2, 3]).send(remote)
         y = torch.FloatTensor([2, 3, 4]).send(remote)
         z = torch.FloatTensor([5, 6, 7]).send(remote)
-        x.get()
-        y.get()
-        z.get()
-        assert (torch.equal(torch.cat([x, y, z]), torch.FloatTensor([1, 2, 3, 2, 3, 4, 5, 6, 7])))
+
+        assert (torch.equal(torch.cat([x, y, z]).get(), torch.FloatTensor([1, 2, 3, 2, 3, 4, 5, 6, 7])))
+
+    def test_local_tensor_multi_var_methods(self):
+        x = torch.FloatTensor([[1, 2], [2, 3], [5, 6]])
+        t, s = torch.max(x, 1)
```

Comment:
Maybe you can now remove the test `test_torch_function_with_multiple_output_on_remote_var` which is not as complete as yours :)
Also, I think `test_torch_function_with_multiple_input_on_remote_var` is also no longer needed too

Reply:
Haha, I'm sorry. I was on an older branch and when I merged to submit a PR I didn't notice your unit tests. I will actually keep those untouched and add some more tests for Variables since you're testing the variables.